src/main/java/frc/robot/constants/ConstRotors.java

```diff
@@ -14,6 +14,8 @@ public class ConstRotors {
   public static final TalonFXConfiguration FUNNEL_BELT_CONFIGURATION = new TalonFXConfiguration();
   public static final double FUNNEL_BELT_SPEED = 0.5;
   public static final double INTAKE_MOTOR_SPEED = 0.5;
+  public static final double COLOR_ROTATING_PANEL_SPEED = 0.3;
+  public static final double TRIPLE_ROTATING_PANEL_SPEED = 0.3;
```

Comment:
Since we decided that the ControlPanel is a motion motor we should move these to constMotion